packages/react-menu/src/components/Menu/Menu.tsx

```diff
@@ -8,7 +8,10 @@ import { useMenuStyles } from './useMenuStyles';
  * Wrapper component that manages state for a popup MenuList and a MenuTrigger
  * {@docCategory Menu }
  */
-export const Menu = React.forwardRef<HTMLElement, MenuProps>((props, ref) => {
+export const Menu: React.FunctionComponent<MenuProps & React.RefAttributes<HTMLElement>> = React.forwardRef<
```

Comment:
Why is this necessary ? 

```suggestion
export const Menu = React.forwardRef<
```

Reply:
I guess this applies for all the changes you made similar to this

Reply:
AFAIR this is to mitigate `tsc` emit lazy type imports. (lazy type imports are not supported by api-extractor ....)

> 🥴 I don't like this TBH as those inlined types are both hard to read and maintain. We should create a post-processing for this to not downgrade DX


**Before:**

```ts
export declare Menu: React.ComponentType<import('@fluentui/react-menu')>.....
```


**After(this change):**
```ts
import {MenuProps} from '@fluentui/react-menu'
export declare Menu: React.ComponentType<MenuProps>.....
```

Reply:
Yes that's the reason for all the similar changes. This is just opinion but I actually like the explicit types on function components (aside from maybe the union on the prop types) since it makes the public-facing API cleaner e.g. when using intellisense as well--probably nobody cares that it's a `ForwardRefExoticComponent` or about a bunch of messy union props.